src/main/resources/kafka-topics.json

```diff
@@ -0,0 +1,9 @@
+{
+  "topics": [
+    {
+      "name": "inventory.instance",
+      "numPartitions": 1,
```

Comment:
Does this mean that the topic is not going to partitioned at the moment? 

I thought there was an intent to partition these messages by instance ID, did I misunderstand?